strings/charoccurrence_test.go

```diff
@@ -0,0 +1,39 @@
+package charoccurrence_test
+
+import (
+	"reflect"
+	"testing"
+)
+
+var testCases = []struct {
```

Comment:
This should probably be moved inside the test function - otherwise it can create conflict.

Reply:
- [x] fixed